### ISSUE TITLE
code corrigibility

### ISSUE THREAD

**Issue by anorak94** [SteffenCzolbe/pyLDDMM#8]
Hey is the code correct because the warp field being generated by the code in the example is not the same as the warp field generated by the image registration ryo-ito github repository you reference in the code. I wrote a simple sum of squared optimization deformation model and that gives almost the same result as the lddmm repository you link in github but youre deformation fields are quite different

**Comment by SteffenCzolbe** [comment]
Hello,
The code was evaluated by visual inspection of results only, no comparison to other implementations has been performed.

Note that deformable image registration is an over-parameterized problem, meaning that there can be infinite deformation fields leading to the same/similar aligned images.

> ... but youre deformation fields are quite different

Different in what way? Do you have examples?

**Comment by anorak94** [comment]
yeah if you look at the repository you link talk about that has a different deformation field. i implemented a solution https://github.com/anorak94/jaxreg/blob/master/reg_ssd_with_smoothing.ipynb. which is just a displacement field optimization using least squares method and it seems to give a similar deformation field as the ryo ito github repsitory you talk about. But i found your code really helpful. i tried to warp a 120x120 circle to square and it doesnt seem to work for me. maybe i am doing something wrong.

**Comment by anorak94** [comment]
https://github.com/Ryo-Ito/image_registration/blob/master/example/LDDMM2d.ipynb
